packages/orchestrator-ui-components/src/pages/startPage/WfoStartPage.tsx

```diff
@@ -90,7 +92,9 @@ export const WfoStartPage = () => {
         listItems:
             processesSummaryResult?.processes.page.map((workflow) => ({
                 title: workflow.workflowName,
-                value: workflow.startedAt,
+                value: parseDateToLocaleDateTimeString(
```

Comment:
Just checking: have you tried `formatDate(...)` -- I think that one does getDate and parseDate together.
Can be found in `date.ts`

Reply:
I tried but not good enough, I throught it did not support the string types, but I retried and works fine. I will update the code.